beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -1217,11 +1217,13 @@ impl<T: BeaconChainTypes> Worker<T> {
                 );
                 self.propagate_validation_result(message_id, peer_id, MessageAcceptance::Reject);
```

Comment:
I think we similarly need to change this to `MessageAcceptance::Ignore` on `BeaconChainError::BlsToExecutionChangeBadFork`

Reply:
updated here: https://github.com/sigp/lighthouse/pull/3880/commits/13196837361d2b4aae404f4fb973720fddc35388